critiquebrainz/frontend/templates/macros.html

```diff
@@ -93,6 +93,19 @@
   </div>
 {%- endmacro %}
 
+{% macro show_tags(tags) %}
+    <div class="tags">
+        {% if tags %}
+          {% for i in range(tags|length - 1) %}
+          <span><a style="font-size:12px" href="{{"https://musicbrainz.org/tag/%s"|format(tags[i]['name'])}}"> {{tags[i]['name']}}, </a></span>
+          {% endfor %}
+          <span><a style="font-size:12px" href="{{"https://musicbrainz.org/tag/%s"|format(tags[tags|length-1]['name'])}}"> {{tags[tags|length -1]['name']}} </a></span>
```

Comment:
Custom styling (like `font-size:12px`) should go into Less files instead of templates.